mainhub/mainhub.ino

```diff
@@ -78,18 +83,29 @@ void draw()
 
 void update_game()
 {
-
+  cursor_index += direction; // Move to the next LED based on the direction
+
+  // Check if the LED reached the end of the strip
+  if (cursor_index >= NUM_LEDS) {
+    cursor_index = NUM_LEDS - 2; // Move to the second-to-last LED
+    direction = -1; // Change the direction to left
+  } else if (cursor_index < 0) {
+    cursor_index = 1; // Move to the second LED
+    direction = 1; // Change the direction to right
+  }
   // Increment the cursor, but reset to zero if it is greater than the number of leds (Circle Path)
-  cursor_index = cursor_index + 1;
+  /*cursor_index = cursor_index + 1;
   if (cursor_index >= NUM_LEDS)
   {
     cursor_index = 0;
   }
-
+*/
```

Comment:
Doesn't look like you need this anymore. Its scary to delete code, but its saved on git so you can always come back to it : https://github.com/LumenFactory/arc-game/commits/main